includes/options/class-amp-options-menu.php

```diff
@@ -318,9 +319,13 @@ public function render_theme_support() {
 						<p><?php esc_html_e( 'Your active theme is known to work well in standard or transitional mode.', 'amp' ); ?></p>
 					</div>
 				<?php endif; ?>
+			<?php elseif ( AMP_Theme_Support::theme_supports_reader_mode() ) : ?>
+				<div class="notice notice-success notice-alt inline">
+					<p><?php echo wp_kses_post( $reader_mode_support ); ?></p>
```

Comment:
`<?php echo` can be simplified to `<?=`:
```suggestion
					<p><?= wp_kses_post( $reader_mode_support ) ?></p>
```
This is safe to use starting from PHP 5.4, as it was decoupled from the [`short_open_tag`](https://www.php.net/manual/en/ini.core.php#ini.short-open-tag) setting that caused conflicts with XML.

Reply:
Ah, great idea.

Reply:
Ah, it would have been better if I simply committed your suggestion here. But I forgot it was there.

https://github.com/ampproject/amp-wp/pull/3188/commits/50c7a14a69663505f6b05fd6f7a7808d19c9f679 applies your suggestion.

Reply:
We don't currently use this shorter syntax anywhere else in the plugin, and it's currently causing a linting failure because we actually currently disallow it it: https://travis-ci.org/ampproject/amp-wp/jobs/582745144#L762-L771

So either we should revert to `<?php echo` or decide to do a mass update to `<?=`. I think we should do the former, and do the latter for the next release, all in one go, and add a PHPCS check to force `<?=` instead of `<?php echo` (old habits die hard!)

Reply:
Yes, agree, let's revert for now.

Reply:
Sure, https://github.com/ampproject/amp-wp/pull/3188/commits/3f2e83d5c1b5e191d08d3d8831fd353e4974b9a8 reverts it. Sorry for the delay.